nucliadb/nucliadb/ingest/consumer/pull.py

```diff
@@ -202,7 +164,7 @@ async def _loop(self):
 
                 except ClientConnectorError:
                     logger.error(
-                        f"Could not connect to {url}, verify your internet connection"
+                        f"Could not connect to processing engine, verify your internet connection"
```

Comment:
It would be nice to keep the url in the log, to know if it's the internal or the public service what we can't connect to